src/app/Dashboard/components/MyActivitiesComponent.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import dateFns from 'date-fns';
+
+import { TableComponent, StatusIndicatorComponent } from '../../common/components';
+import TruncateDescriptionComponent from './TruncateDescriptionComponent';
+
+const MyActivitiesComponent = (props) => {
+  const { userActivities } = props;
+  const columnNames = ['Activity', 'Date', 'Description', 'Points', 'Status'];
+  let tableBodyHtml;
+  if (!userActivities.length) {
+    tableBodyHtml = (
+      <tr className='myactivities__table__row'>
+        <td />
+        <td />
```

Comment:
What is going on here?

Reply:
I have rectified this. Added `colspan` attribute.